config/ulib_config.h

```diff
@@ -53,6 +53,29 @@ extern "C" {
  */
 #define AZ_ULIB_CONFIG_MAX_LOG_SIZE        256
 
+/**
+ * @brief   IPC public API shall validate the contract
+ *
+ *  This definition enables the argument check on all IPC public APIs. This check will verify if the
+ *      provided argument follow the contract defined in the documentation. 
+ *  Commenting this definition, the IPC public APIs will not test any of the received arguments.
+ */
+#define AZ_ULIB_CONFIG_IPC_VALIDATE_CONTRACT
+
+/**
+ * @brief   Maximum number of interfaces published in the IPC.
+ *
+ *  Defines the maximum number of interfaces that the system can publish on the IPC.
+ */
+#define AZ_ULIB_CONFIG_MAX_IPC_INTERFACE   10
+
+/**
+ * @brief   Maximum interface instances.
```

Comment:
>Maximum interface instances. [](start = 12, length = 28)

can there be a further description of what this means? is this the result of a `get_interface` call?